tests/unit/amo/components/TestAddonVersionCard.js

```diff
@@ -163,18 +163,6 @@ describe(__filename, () => {
         })}`,
       );
     });
-
-    it('renders nothing for released date and file size when no file exists for the version', () => {
-      const version = { ...fakeVersion, files: [] };
-      const addon = { ...fakeAddon, current_version: version };
-      _loadVersions({ addon });
-
-      const root = render({
-        version: createInternalVersionWithLang(version),
-      });
-
-      expect(root.find('.AddonVersionCard-fileInfo')).toHaveLength(0);
-    });
```

Comment:
Why is this test case removed?

Reply:
because versions can't be a state where no file exists

Reply:
What changed in addons-server to enable this?

If we switched from a potentially-empty array to a single field, it does not tell me how this single field is always defined (a.k.a. not null) so I'd expect possibly null values, unless something else has changed to get rid of versions without any file.

Reply:
mozilla/addons-server#17839 which was the api follow-up to mozilla/addons-server#16465 and mozilla/addons-server#16464 dealt with the existing data.